{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly.
+
+### PR Cover Letter
+- [ ] Include A detailed description of the changes in the body of the PR
+
+### Testing
+- [ ] Unit, integration, acceptance tests as appropriate
+- [ ] Database migrations are backwards-compatible
+
+### DevOps assistance
+If it's needed, or even if you suspect it might be needed, do any/all these
+things as early as possible. Don't surprise devops right before merging!
+For example, touching envs/aws.py or or envs/common.py are dead giveaways
+that you'll need assistance when pushing your changes out.
+
+Things that require devops attention include, but are not limited to:
+- configuration changes
+- feature flags
+- migrations
+
+Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
+- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
+- [ ] File a support ticket
+
+FYI: Tag anyone who might be interested in this PR here.
+
+### Post-review
+- [ ] Squash commits
+
+## Reviewers
+If you've been tagged for review, please check your corresponding box once you've given the :+1:.
+- [ ] Code review #1
+- [ ] Code review #2
+- [ ] Docs review (required for UI strings/error messages)
+- [ ] UX review
+- [ ] Accessibility review
+- [ ] Product review
+
+### Areas to Consider
+- [ ] i18n 
+    - Are all user-facing strings tagged?
+- [ ] Right-to-left
+    - Will the feature work for right-to-left languages?
+- [ ] Analytics
+    - Are any new events being emitted?
+    - Have any events been changed?
+    - Are there any new user actions that should be emitted as events?
+- [ ] Performance
+    - What dimensions does this change scale over? Users? Courses? Course size?
+    - How does the feature scale over those dimensions? Sub-linear? Linear? Quadratic? Exponential?
+- [ ] Database migrations
+    - Are they backwards compatible?
+    - When they run on production, how long will they take? Will they lock the table?
+
+### Documentation considerations
+- [ ] Are there user visible changes? If so, list who is affected, and include a pointer to the relevant code.
+- [ ] Should this be mentioned in the release notes? If so, make sure that you included a description.
```

Comment:
I'd actually love to get to a point where what we commit to is the git commit messages being the release notes. They are intended to be structured in a high-level-subject/detailed-body format that I think would work nicely in the notes, and I think having them as the release notes would incentivise us to actually put meaningful messages in the commits.

The automated compilation process could then give us the option of editing them as needed.